.mage/makefile.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"go.einride.tech/mage-tools/mgmake"
+	"go.einride.tech/mage-tools/mgpath"
+)
+
+func main() {
+	mgmake.GenerateMakefiles(
```

Comment:
I wish we could solve this in some other way, the only reason I had to go for this approach is because we need to set the global variable which this method sets in order to know which makefikes to generate.

open to suggestions, can we solve this in some other way? 🤔

also maybe we should sanity check this file, since it’s ignored later when we compile, should we make sure there are no other methods or something?